src/spectrecoff-cli/commands/BarChart.fs

```diff
@@ -35,4 +35,36 @@ type BarChartDocumentation() =
 
     override _.Execute(_context, _settings) =
         Cli.Theme.setDocumentationStyle
+        NewLine |> toConsole
+        pumped "BarChart submodule"
+        |> alignedRule Left
+        |> toConsole
+
+        Many [
+            CO [
+                C "This submodule provides functionality from the BarChart widget of Spectre.Console ("
+                Link "https://spectreconsole.net/widgets/barchart"
+                C ")"
+            ]
+            NL
+            C "The bar chart can be used using the barChart function:"
+            BI [
+                P "barChart: string -> ChartItem list -> OutputPayload"
+            ]
+            NL
+            CO [C "The"; P "ChartItem"; C "union type consists of two options:"]
+            BI [
+                CO [P "ChartItem:"; C "Consists of the label and a value for the item."]
+                CO [P "ChartItemWithColor:"; C "Additionally defines a color the item should be rendered in."]
```

Comment:
Maybe nitpicky, but I'd prefer direct language. I mean, having an api that sais something _should be rendered green_ is nice, but as a user I'd prefer _it will be rendered green_ ;)